crates/freya/src/_docs/hot_reload.rs

```diff
@@ -7,14 +7,18 @@
 //! Before launching your app, you need to initialize the hot-reload context:
 //!
 //! ```rust, no_run
-//! use freya::prelude::*;
-//! use freya::hot_reload::FreyaCtx;
+//! # use freya::prelude::*;
```

Comment:
I would actually leave this one visible, it makes more sense to me
```suggestion
//! use freya::prelude::*;
```